tsconfig.json

```diff
@@ -14,10 +16,9 @@
     "suppressImplicitAnyIndexErrors": true,
     "strictNullChecks": true,
     "noFallthroughCasesInSwitch": true,
-    "noUnusedParameters": true,
+    "noUnusedParameters": false,
```

Comment:
because so often we have `(req, res, next)` where `res` isn't used

Reply:
Can we configure it like ESLint using these rules rather than disabling it outright? I'm not sure how flexible it is in this regard.

1. we don't care if parameters start with underscores (e.g. `_res` would be ignored)
2. we don't care if one of the parameters *after* this one is used (e.g. `(req, res, next) => next()` shouldn't raise)

Reply:
ah, perfect. i wasn't aware that that was a thing.  looks like ts honors it as well. 2396b0e